rdiodj/static/js/chat.js

```diff
@@ -1,236 +1,188 @@
 var chat = chat || {};
-
-chat.User = Backbone.Model.extend({
-
-});
-
+chat.User = Backbone.Model.extend({});
 chat.firebasePeopleRef = new Firebase(firebaseRootUrl + '/people');
 chat.firebaseMessagesRef = new Firebase(firebaseRootUrl + '/messages');
-
 chat.UserList = Backbone.Firebase.Collection.extend({
-  model: chat.User,
-
-  firebase: chat.firebasePeopleRef, // pass the ref here instead of string so we can listen for disconnect.
-
-  getOnlineUsers: function() {
-    return this.filter(function(user) {
-      return user.get('isOnline');
-    });
-  }
+	model: chat.User,
+	firebase: chat.firebasePeopleRef,
+	// pass the ref here instead of string so we can listen for disconnect.
```

Comment:
I'll make the change. I'm going to submit another commit to add in comments to my code.